### DISCUSSION TITLE
Tò mò về nguồn tài trợ của các bạn

### DISCUSSION THREAD

**Discussion by ooker777** [undertheseanlp/underthesea#683] [General]
Nếu các bạn thoải mái thì có thể chia sẻ chút về nguồn tiền của các bạn đến từ đâu không? Mình cũng muốn kiếm tiền làm dự án nên muốn học hỏi các bạn :D

**Comment by rain1024** [comment]
@ooker777

> Nếu các bạn thoải mái thì có thể chia sẻ chút về nguồn tiền của các bạn đến từ đâu không?

underthesea không có nguồn tài trợ cố định bạn nhé. Underthesea có kêu gọi donate nhưng cũng không được bao nhiêu.

>  Mình cũng muốn kiếm tiền làm dự án nên muốn học hỏi các bạn :D

bạn muốn kiếm tiền thì nên học hỏi dự án khác, ví dụ [langchain](https://blog.langchain.dev/announcing-our-10m-seed-round-led-by-benchmark/) chẳng hạn :D

**Reply by ooker777** [reply]
các bạn đều làm vào những lúc rảnh hết à? Nhóm có ai là làm toàn thời gian không? Các bạn đã thử gọi vốn đầu tư chưa?

**Reply by rain1024** [reply]
> các bạn đều làm vào những lúc rảnh hết à?

đúng rồi bạn

> Nhóm có ai là làm toàn thời gian không? 

không bạn nhé

> Các bạn đã thử gọi vốn đầu tư chưa?

không bạn nhé

**Reply by ooker777** [reply]
Các bạn chỉ làm vì vui thôi chứ không có ý định kiếm thêm thu nhập từ nó à? Mình có thể hỏi vì sao ko?
